stl/inc/chrono

```diff
@@ -2215,6 +2215,36 @@ namespace chrono {
         return hours{_Ret};
     }
 
+    // [time.zone.timezone]
+
+    // TRANSITION: work in progress
+    class time_zone {
+    public:
+        time_zone(std::string_view _Name) : _Name(_Name) {}
+
+        // FIXME: The standard denotes move semantics are required. Is is valid to add
+        //        copy semantics as well to make interacting with the CRT allocator easier.
+        // time_zone(time_zone&&) = default;
+        // time_zone& operator=(time_zone&&) = default;

```

Comment:
No, we generally believe that copyability is observable so we don't add it. Why would being copyable make things easier?

Reply:
This had nothing to do with the CRT allocator. The copy was not complicated, I just needed to use `std::transform` to copy the `time_zone`/`time_zone_link` vectors.